utils/encryption.js

```diff
@@ -0,0 +1,16 @@
+const bcrypt = require('bcrypt')
+const saltRounds = 10
```

Comment:
suggestion: setup a prettier config file and have vscode format on save https://prettier.io/docs/en/configuration.html